### PR TITLE
Use agreed upon Twig standards

### PR DESCRIPTION
**Pull request by BPScott**
* a space around the contents in {{ }} braces
* use underscores in variable names

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: app/Resources/views/find_by_pid/programme.html.twig [hunk 1]
```diff
@@ -6,10 +6,10 @@
...
     <div class="br-box-page">
         <h1>It Lives!</h1>
-        {{ds2013('programme', programme, {
+        {{ ds2013('programme', programme, {
             'showSynopsis': true
-        })}}
+        }) }}
 
-        {{dump(programme)}}
+        {{ dump(programme) }}
```
Comment (votemike) [inline app/Resources/views/find_by_pid/programme.html.twig]: Should this be here?

Comment (BPScott) [inline app/Resources/views/find_by_pid/programme.html.twig]: yeah, the findbypid page isn't done yet, but I wanted to prove the concept of how to pass things around. dump was the easiest way of proving things are were spat out.

**Comment by votemike** [comment]
Looks OK to me...

File: app/Resources/views/home/show.html.twig [hunk 2]
```diff
@@ -29,16 +29,16 @@
...
 {% block body %}
-<h1 class="visually-hidden">{{tr('programmes')}}</h1>
+<h1 class="visually-hidden">{{ tr('programmes') }}</h1>
 
 <div class="delta no-margin">
     <div class="br-box-highlight">
         <div class="programmes-page b-g-p home-intro">
-            <p class="alpha deemphasize text--center"><span data-count="{{programmeCount}}">{{programmeCount|number_format}}</span> total programmes & groups</p>
-            <div class="search"><form action="{{path('atoz_list_all')}}" method="get">
+            <p class="alpha deemphasize text--center"><span data-count="{{ programme_count }}">{{ programme_count|number_format }}</span> total programmes & groups</p>
```
Comment (votemike) [inline app/Resources/views/home/show.html.twig]: Does the controller not also need to be altered?

Comment (BPScott) [inline app/Resources/views/home/show.html.twig]: It has been, see HomeController.php.

Comment (votemike) [inline app/Resources/views/home/show.html.twig]: Yep. I wasn't paying attention...

**Pull request merged**